build.gradle

```diff
@@ -272,3 +274,10 @@ task run(type: Run) {
   group = 'Verification'
   impliesSubProjects = true
 }
+
+task buildSrcIntegTest(type: GradleBuild) {
```

Comment:
Adding `integTest` as a dependency of `check` in `buildSrc` would mean that 
```
git checkout some_branch
gradle assemble
```
would execute the integration test in buildSrc as `buildSrc:check` is invoked by default.